examples/dh-5-example/build.gradle

```diff
@@ -8,53 +9,12 @@ buildscript {
         if (project.hasProperty("testing")) {
             classpath "com.marklogic:ml-data-hub:5.3-SNAPSHOT"
         } else {
-            classpath "gradle.plugin.com.marklogic:ml-data-hub:5.2.0"
+            classpath "gradle.plugin.com.marklogic:ml-data-hub:5.3.0"
         }
     }
 }
-
 plugins {
     id "net.saliman.properties" version "1.5.1"
     id "java"
 }
-apply plugin: "com.marklogic.ml-data-hub"
-
-repositories {

```

Comment:
We still need everything that's been deleted here - extractZip is needed to make sample data available, and runFlowWithoutProject is referenced in the README file.